### DISCUSSION TITLE
message->contact() is null, why?

### DISCUSSION THREAD

**Discussion by ChillMouse** [def-studio/telegraph#315] [Q&A] [answered]
Hello, my bot can handle my messages with my contact is correct. But another users, after send contact, getting error. 

In logs like: "->phoneNumber() call in null"

And in debug log webhook: "{'text': '78889998876'}"

I have question.. why contact is null?

**Comment by ChillMouse** [comment, marked as answer]
Okay, fixed it.

Need everytime check $this->message->contact() is eq. "null" or no

But doesn't works option of command drop pending "php artisan telegraph:unset-webhook --drop-pending-updates"

**Reply by fabio-ivona** [reply]
fixed in latest release
